client/my-sites/plugins/plugin-activate-toggle/index.jsx

```diff
@@ -97,12 +100,15 @@ PluginActivateToggle.propTypes = {
 };
 
 PluginActivateToggle.defaultProps = {
+	hideOnAutomatedTransfer: false,
 	isMock: false,
 	disabled: false,
 };
 
 export default connect(
-	null,
+	( state, { plugin, site } ) => ( {
+		hideOnAutomatedTransfer: isSiteAutomatedTransfer( state, site.ID ) && includes( [ 'jetpack', 'vaultpress' ], plugin.slug ),
```

Comment:
Let's be safe accessing ID off of site by using lodash get. Same thing for plugin.slug.